### PR TITLE
Addressing reviewer comments

### PR DESCRIPTION
**Pull request by lexnederbragt**

### CODE REVIEW

File: ten_quick_tips_live_coding.md [hunk 1]
```diff
@@ -56,6 +56,14 @@ instructors will often make unplanned mistakes.
 Novice learners are likely to make many such mistakes themselves,
 and diagnosing and solving mistakes is an integral aspect of learning programming.
 
+The participatory, 'code-along' aspect
```
Comment (raynamharris) [inline ten_quick_tips_live_coding.md]: A few different points are being made here: the target audience (novice, experienced) matters, how they engage (passive versus active) matters. Suggest rewording and elaborating. Something like... 

The participatory aspect engages learners, which helps them become active practitioners rather than passive observers of the programming process. Participatory live coding is most beneficial for novices who are unfamiliar with the tools. More experienced learners may gain enough by listening passively or engaging with the material and classroom differently.

File: ten_quick_tips_live_coding.md [hunk 2]
```diff
@@ -105,14 +115,15 @@ or who may have hearing, vision, or mobility impairments.
 If the output of your command or code makes what you just typed disappear from view,
 scroll back up so learners can see it again.
 
-If your current practice involves copying and pasting code
-from your lesson material, try transitioning to a technique where you,
+Do not ask your learners to copy and past code or commands
```
Comment (raynamharris) [inline ten_quick_tips_live_coding.md]: I like to steer clear of the black and white "do not" because there are times when a piece of a command is shared (like a URL or a path) and offer an alternative instead. Like "When teaching,  you frequently copy and paste commands from a tutorial into the console or use pre-prepared notebooks, we recommend transitioning to a strategy where you build the scripts one command at a time with the learners. Learners can copy and paste commands from if they need to catch up or are practicing asynchronously with an online tutorial."

**Review by raynamharris — CHANGES_REQUESTED** [review]
I made a few comments, but the rest of the additions read well to me.

File: ten_quick_tips_live_coding.md (hunk 2 above)
Comment (lexnederbragt) [reply ten_quick_tips_live_coding.md]: This change is the result if the following reviewer comment:

> [this text] would be stronger by starting the paragraph with the actual advice (don't copy & paste/that copy-paste is different than participatory live coding) and includes the compromise in the paragraph, rather than leading in with a compromise and then situating it.

The change reverses this order. Your proposal would reverse it back.

I do, however, agree with the 'steer clear of the black and white "do not"', and made a suggestion to solve it, and incorporated your second sentence.
